docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -89,3 +444,14 @@ However, this script requires administration access to the database. Depending o
 The :command:`grant_bareos_privileges` command is required, if new databases tables are introduced. It does not hurt to run it multiple times.
 
 After this, restart the Bareos Director and verify it starts without problems.
+
+
+.. warning::
+
+   If you're updating from Bareos <= 13.2.3 and have configured the Bareos database during install using Bareos environment variables
+   (``db_name``\ , ``db_user``\  or ``db_password``\ , see :ref:`CatMaintenanceChapter`), make sure to have these variables defined
+   in the same way when calling the update and grant scripts. Newer versions of Bareos read these variables from the Director configuration.
+   However, make sure that the user running the database scripts has read access to this file (or set the environment variables).
+   The **postgres** user normally does not have the required permissions.
```

Comment:
Can we drop this part? I think bareos 13 is really too old.

Reply:
I'm in favor of. 
Thanks for having done this review.

Reply:
Done.